product_attribute_code/product_view.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Module for OpenERP
+  Copyright (C) 2015 Akretion (http://www.akretion.com). All Rights Reserved
+  @author Benoît GUILLOT <benoit.guillot@akretion.com>
+  The licence is in the file __openerp__.py
+-->
+
+<openerp>
+    <data>
+        <!-- INHERITED VIEW FOR THE OBJECT : product_attribute -->
+
+        <record id="attr_tree_view" model="ir.ui.view">
+            <field name="name">product_attribute_code.attribute.tree</field>
+            <field name="model">product.attribute</field>
+            <field name="inherit_id" ref="product.attribute_tree_view"/>
+            <field name="arch" type="xml">
+                <field name="name" position="before">
+                    <field name="sequence"/>
```

Comment:
consider adding widget="handle" to this tag.  Unless there is a really good reason for exposing sequence numbers to users try to avoid.  Have tested and works well in editable tree view